### PR TITLE
Swerve Subsystem Javadoc Entries

### PR DESCRIPTION
**Pull request by ddjoshua01**
Wooo comments

### CODE REVIEW

**Review by acuaroo — APPROVED** [review]
Fine, but remember comments shouldn't just describe the code, it should give a why. 

You've done this in some places, but in others, comments are just redundant.

**Review by aaditsangvikar — CHANGES_REQUESTED** [review]
You changed some telemetry code...? Not sure if this was an accident, but in the setting of network tables odometry, you switched a frontRight to a frontLeft. 

Relating to the comments, they look pretty good, but you don't need to switch stuff like "Set Network Tables Telemetry" to "Set telemetry in network tables".

I think it would be best if you left the previous comments that were already there untouched, and added your own descriptions below them, or something like that.

**Comment by ddjoshua01** [comment]
oh ok

File: src/main/java/frc/robot/subsystems/SwerveSubsystem.java [hunk 1]
```diff
@@ -170,24 +174,28 @@ private void resetOdometry(Pose2d pose) {
 
     // Drive function - slew rate limited to prevent shearing of wheels
     public void drive(double forwardMetersPerSecond, double sidewaysMetersPerSecond, double radiansPerSecond, boolean fieldRelative, boolean rateLimit) {
```
Comment (TotalTaxAmount) [inline src/main/java/frc/robot/subsystems/SwerveSubsystem.java]: Above all functions please add a javadoc entry so in the future we can generate readable documentation for out code. 
[How to write javadoc](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html)

**Review by TotalTaxAmount — CHANGES_REQUESTED** [review]
Looks good, just add the javadoc  entries for the functions

**Review by acuaroo — CHANGES_REQUESTED** [review]
Looks good! Just add @return to the Javadoc stuff when it's not void.

**Pull request merged**